src/requirements.yml

```diff
@@ -1,3 +1,5 @@
+# - src: https://github.com/cisagov/ansible-role-amazon-efs-utils
+#   name: amazon_efs_utils
```

Comment:
What's going on here?

Reply:
What he ⬆️ said.  If we don't need the EFS utils here, shouldn't we just delete it from this file and `src/aws.yml`?

Reply:
My thought was that EFS is something that is likely to be be required later once users realize that perhaps it was foolish to ask that it be removed.  I am happy to remove the code, if you guys still think that it wise.  Let me know.

Reply:
This is the FreeIPA server - aren't we (the Dev team) the primary users?  If we think we need EFS here, let's include it.  Do we have a use case for it?

Reply:
Ah, OK.  I was operating on autopilot and not paying attention to which repo this is.

Reply:
See commit 5026358.